### PR TITLE
fix: Fixed triggering of workflows due to `labeling` of the PR's

### PR DESCRIPTION
**Pull request by Sai-Suraj-27**
# PR Description
Fixed triggering of workflows due to `labeling` of the PR's to save compute time.

## Related Issue
Closes #

## Checklist

- [ ] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [ ] Did you follow the steps we provided?

### CODE REVIEW

File: .github/workflows/dockerfile-image.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ "main" ]
   pull_request:
-    types: [labeled, review_requested]
```
Comment (Sai-Suraj-27) [inline .github/workflows/dockerfile-image.yml:7]: @vedpatwardhan I'm not sure about this one.

Comment (vedpatwardhan) [reply .github/workflows/dockerfile-image.yml:7]: Yeah I think we can remove it from here as well 👍

**Review by vedpatwardhan — APPROVED** [review]
lgtm! Feel free to merge the PR, thanks @Sai-Suraj-27 😄

File: .github/workflows/dockerfile-image.yml (hunk 1 above)
Comment (Sai-Suraj-27) [reply .github/workflows/dockerfile-image.yml:7]: Sure 👍🏻

**Pull request merged**